internal/services/digitaltwins/digital_twins_instance_resource_test.go

```diff
@@ -181,3 +215,30 @@ resource "azurerm_digital_twins_instance" "test" {
 }
 `, r.template(data), data.RandomInteger)
 }
+
+func (r DigitalTwinsInstanceResource) identity(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_digital_twins_instance" "test" {
+  name                = "acctest-DT-%d"
+  resource_group_name = azurerm_resource_group.test.name
+  location            = azurerm_resource_group.test.location
+  identity {
+    type = "SystemAssigned"
+  }
+}
+`, r.template(data), data.RandomInteger)
+}
+
+func (r DigitalTwinsInstanceResource) identityNone(data acceptance.TestData) string {
```

Comment:
we can use `basic` for this purpose?

Reply:
@tombuildsstuff yes, I've updated the test code to reuse the basic config